MdeModulePkg/Universal/Variable/RuntimeDxe/Variable.c

```diff
@@ -283,6 +283,53 @@ UpdateVariableStore (
   return EFI_SUCCESS;
 }
 
+// MU_CHANGE Starts: Deleted hw error records should not count towards the hw error record space quota
+///
+/// The purpose of this routine is to make the deleted HwErrRec to be counted against CommonVariable quota
+/// This is routine is to be called upon deletion of HwErrRec variables and initialization
+///
+STATIC
+VOID
+DeletedHwErrRecToCommonQuota (
+  VARIABLE_HEADER  *VarHeaderPtr
+  )
+{
+  VARIABLE_HEADER  *tNextVariable;

```

Comment:
The `t` prefix names are against convention, but don't need to be cleaned up here if you don't care to.

Reply:
No it looked weird to me - but I wasn't sure if it was intentional or not. I'll go ahead and fix it. I'm not quite confident what the 't' was supposed to represent